tests/test_int_bitwise.nim

```diff
@@ -257,29 +100,6 @@ template testBitwise(chk, tst: untyped) =
     chkShl(chk, "0F", 255, "8000000000000000000000000000000000000000000000000000000000000000", 256)
 
   tst "operator `shr`":
-    chkShr(chk, "0F", 4, "00", 8)
```

Comment:
why the removal of these tests?

Reply:
they are for stuint[8-32], I don't think they are used.